### PR TITLE
feat:create models users

### PR DESCRIPTION
**Pull request by LeVHoangduc**
@lenguyenthimy-agilityio @hungnguyenhuu-agilityio Please help me to check and merge it

### CODE REVIEW

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu-agilityio I have updated it. Please help me to check again.

File: src/javascripts/models/user.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+   * Method to get list of users from service
+   * @returns {Object} userList object
+   */
+  getUserList = async () => {
+    this.userList = await this.service.getUserList();
+    return this.userList;
+  };
+}
+
+export default Users;
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/models/user.js]: Should think about the flow of your app.

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu-agilityio I have updated my user model. Please help me to check again.

File: src/javascripts/models/user.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import ApiService from "../services/apiService";
+class User {
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/models/user.js:3]: code style

File: src/javascripts/models/user.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+   * Method to check user current is valid
+   * @param {Object} userCurrent
+   * @returns {Boolean} is exist user
+   */
+  isValidUser = async (userCurrent) => {
+    const userList = await this.getUserList();
+    const isValidUSer = userList.find((user) => {
+      return user.username === userCurrent.username && user.password === userCurrent.password ? true : false;
+    });
+    return isValidUSer;
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/models/user.js:30]: code style

File: src/javascripts/models/user.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  /**
+   * Function to get list of users from service
+   * @returns {Object} userList object
+   */
+  getUserList = async () => {
+    // service for fetch data
+    this.userList = await this.service.getList("/users");
+    return this.userList;
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/models/user.js:16]: code style

File: src/javascripts/models/user.js (hunk 3 above)
Comment (LeVHoangduc) [reply src/javascripts/models/user.js:30]: @hungnguyenhuu-agilityio  the code line is long?

**Comment by hungnguyenhuu-agilityio** [comment]
Conflict. Pls check

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu-agilityio I have updated it. Please help me to check again

File: src/javascripts/models/user.js (hunk 2 above)
Comment (hungnguyenhuu-agilityio) [reply src/javascripts/models/user.js:3]: Seem like you missed that @LeVHoangduc

Comment (LeVHoangduc) [reply src/javascripts/models/user.js:3]: @hungnguyenhuu-agilityio I have updated it.

**Pull request merged**